integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -1058,6 +1110,11 @@ private void initialize(Map<String, Object> inputDomainMap) throws Exception {
       domainMap.put("t3PublicAddress", TestUtils.getHostName());
     }
 
+    domainMap.put("domainHome", "/shared/domains/" + domainUid);
```

Comment:
You don't need the next 5 lines. Its done in line 1129 - 1136.

Reply:
Done